src/alp/backend/common.py

```diff
@@ -0,0 +1,31 @@
+import hashlib
+import json
+
+
+def create_model_hash(model, batch_size):
+    # convert dict to json string
+    model_str = json.dumps(model)
+
+    # create the model hash from the stringified json
+    mh = hashlib.md5()
+    str_concat_m = str(model_str) + str(batch_size)
+    mh.update(str_concat_m.encode('utf-8'))
+    return mh.hexdigest()
+
+
+def create_data_hash(data):
+    for i in range(len(data)):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using enumerate instead of iterating with range and len](https://www.codacy.com/app/thomas-boquet/python-alp/file/2724954331/issues/source?bid=3232749&fileBranchId=3348372#l17)